### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by prath-dev**
Screenshot from 2022-02-22 01-02-30

### CODE REVIEW

File: Views/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pdui.netlify.app/pdui.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <link rel="stylesheet" href="..//Components/Css/wishlist.css">
+    <title>Document</title>
```
Comment (05him) [inline Views/wishlist.html:11]: change title

File: Views/signup.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pdui.netlify.app/pdui.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <link rel="stylesheet" href="..//Components/Css/signup.css">
+    <title>Document</title>
```
Comment (05him) [inline Views/signup.html:11]: change title

**Review by 05him — APPROVED** [review]
Eyerything else looks good to me , just one more suggestion try using 
body{
margin:0;
padding:0;
}

this will prevent the default padding and margin on the body and will make content end to end

File: Components/Css/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.Wishlist-header {
+    display: flex;
+    justify-content: center;
+    color: darkblue;
+}
+
+.vertical-card {
+    border: 2px solid blue;
+}
```
Comment (Samarth-5109) [inline Components/Css/wishlist.css:9]: use CSS variable instead of blue, darkblue

**Review by Samarth-5109 — APPROVED** [review]

**Comment by prath-dev** [comment]
@Samarth-5109 @05him Thanks for review i will do changes in second version of my ecommerce ..

File: Views/signup.html (hunk 2 above)
Comment (vidhi-thakur) [reply Views/signup.html:11]: Yes, you can update the title to `sign up` form or something

File: Views/signup.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        <input class="search-bar" type="text" placeholder="Search">
+        <a href="..//Views/cart.html">Cart<span class="material-icons-outlined">
+            shopping_cart
+          </span>
+              </a>
+        <a href="..//Views/wishlist.html">WishList<span class="material-icons-outlined">
+       favorite_border
+        </span>
+          </a>
+        <a href="..//Views/login.html">Login</a>
```
Comment (vidhi-thakur) [inline Views/signup.html:26]: I believe only one `/` should be in the path of href link, please check once. Kn other anchor tags as well

**Review by vidhi-thakur — APPROVED** [review]
Added some suggestions, please check

File: Views/signup.html (hunk 4 above)
Comment (prath-dev) [reply Views/signup.html:26]: okkk i will check ..

**Review by shivamSspirit — COMMENTED** [review]
good look to me

**Pull request merged**